package.json

```diff
@@ -85,7 +85,7 @@
     "docs": "typedoc",
     "lint": "eslint src test --ext .js,.ts && tsc",
     "test": "mocha --forbid-only test/unit test/unit/token test/unit/tracking-buffer",
-    "test-integration": "mocha --forbid-only test/integration/",
+    "test-integration": "mocha test/integration/",
```

Comment:
Hi @arthurschreiber , is this forbid-only removed by an accident or it will inference with the new configuration handle for integration test? Or just for easy debugging?

Reply:
Ah, no, that's a mistake and needs to be reverted before this can be merged. Good catch!